readthedocs/doc_builder/environments.py

```diff
@@ -658,7 +658,7 @@ def failed(self):
     def done(self):
         """Is build in finished state."""
         return (
-            self.build is not None and
+            self.build and
```

Comment:
Why was this changed?

Reply:
Here we initialize build as an empty dict which is not equal to `None` and it raises some errors while calling `successful` as `self.build is not None`  returns `True` and tries to get the key `state`
https://github.com/readthedocs/readthedocs.org/blob/801f63be924266af69a6bd4a9234a8010ebcb4b1/readthedocs/projects/tasks.py#L322